.github/workflows/deploy-sdk-infra.yaml

```
@@ -0,0 +1,149 @@
+              "title":"Link to GitHub Action: ${{github.run_id}}",
+              "url":"https://github.com/${{github.repository}}/actions/runs/${{github.run_id}}"
+            }]
+        with:
+          args:
+            "${{ github.event.inputs.environment == 'production' && 'PRODUCTION' || 'STAGING' }}:
+            SDK_INFRA deployment `${{ github.run_id }}` started 🚀 (from branch `${{ github.ref_name
+            }}` by [${{ github.triggering_actor }}](https://github.com/${{ github.triggering_actor
+            }}))."
+
+      - name: Git clone the repository
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)